jerry-core/ecma/builtin-objects/ecma-builtin-global.c

```diff
@@ -27,6 +27,7 @@
 #include "lit-magic-strings.h"
 #include "lit-strings.h"
 #include "vm.h"
+#include "jcontext.h"
```

Comment:
This include looks unnecessary to me.

Reply:
It's necessary since the `ECMA_GET_SUPER_EVAL_PARSER_OPTS` macro references to the `JERRY_CONTEXT(status_flags)` field.